R/delineate.R

```diff
@@ -74,27 +74,31 @@ delineate <- function(
     force_download = force_download
   )
 
-  # Get the bounding box and (if not provided) the CRS
-  if (is.null(crs)) crs <- get_utm_zone(osm_data$aoi)
+  # If not provided, determine the CRS
+  if (is.null(crs)) crs <- get_utm_zone(osm_data$bb)
```

Comment:
The `aoi` here could not be present if corridor is FALSE. We can re-add the `bb` to be returned by `get_osm_data`, shouldn't hurt.